### DISCUSSION TITLE
Front end app notifications

### DISCUSSION THREAD

**Discussion by looped-monk** [saleor/saleor#10599] [General]
I have a dilemma : 

I'm preparing a simple flutter ecommerce with blogging app and connect to the Graphql api. My only dilemma is how do i setup push notifications for my flutter  app which sends important notifications to user ?  From my understanding,  I need to use webhooks and deliver the notification to my end user when the event is pushed by saleor core to the created dashboard app, the endpoint of which will be available for connecting to front end ?

Regarding user synch, I will be sending the jwt payload with information to connect to Saleor docker from my django blogging backend api.

**Comment by peelar** [comment]
Hello @narayanan-ka 👋 

The solution you described sounds about right. Please don't hesitate to reach out when you encounter any issues with Saleor during the development. Good luck! 🤞

**Reply by looped-monk** [reply]
Sure @peelar will do. Also I'm currently conversing with you regarding Saleor as a marketplace in another thread. Hoping to receive a response there.
